react/src/pages/ServingListPage.tsx

```diff
@@ -280,7 +283,8 @@ const ServingListPage: React.FC<PropsWithChildren> = ({ children }) => {
               url
               open_to_public
               created_at @since(version: "23.09.0")
-              created_user
+              created_user_email @since(version: "23.09.8")
+              session_owner_email @since(version: "23.09.8")
```

Comment:
Same here: We can remove this because these fields aren't used in RoutingListPage. It is used EndpointOwenInfo.
```suggestion

```